test/pkg/environment/monitor.go

```diff
@@ -81,6 +81,15 @@ func (m *Monitor) RestartCount() map[string]int {
 	return restarts
 }
 
+// GetNodes returns the most recent recording of nodes
+func (m *Monitor) GetNodes() []v1.Node {
+	m.poll()
+	m.mu.RLock()
+	defer m.mu.RUnlock()
+	last := m.recordings[len(m.recordings)-1]
```

Comment:
any reason to not inline?

Reply:
Thought the same, ultimately decided just to follow the style of other functions in this file.